src/Compilers/VisualBasic/Test/Syntax/Parser/ParseDirectives.vb

```diff
@@ -21,6 +21,35 @@ Public Class ParseDirectives
         ]]>)
     End Sub
 
+    Public Sub ParseShebangDirective()
+        ParseAndVerify("#!", TestOptions.Script)
+        ParseAndVerify("#!/usr/bin/env csharp", TestOptions.Script)
```

Comment:
It doesn't affect the test, but it might be more appropriate to use "visualbasic" or "vb".